app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ApplicationsController < ApplicationController
+
+  def show
+    @application = Application.find(params[:id])
+    @application_pets = @application.pets
+    @pets = []
+    
+    if params[:search].present?
+      @pets = Pet.search(params[:search])
+      # require 'pry';binding.pry
```

Comment:
We'll wanna clean up our prys lying around eventually, but we may need this one still.